projboard/tests/test_login.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+from django.contrib.auth.models import User
+from pytest_django.asserts import assertTemplateUsed
+
+VALID_USER = "smarticleUser"
+VALID_EMAIL = "user@telhai.ac.il"
+VALID_PASSWORD = "smarticlePassword"
+
+INVALID_EMAIL = "user.telhai.ac.il"
+INVALID_PASSWORD_MISMATCH = "smarticlePasswordMismatch"
+LOGIN_PATH = "login/login.html"
```

Comment:
I suggest moving all the defined constants to `conftest.py`, because I will use them also for testing `signup` feature